Mesh_3/doc/Mesh_3/CGAL/refine_mesh_3.h

```diff
@@ -1,165 +1,5 @@
-to set the parameters of this optimizer.
-If one parameter is not set, the default value of
-`odt_optimize_mesh_3()` is used for this parameter.
-
-- <b>`perturb`</b> `parameters::perturb()` and `parameters::no_perturb()` are designed to
-trigger or not a call to `perturb_mesh_3()` function and
-to set the parameters of this optimizer. If one parameter is not set, the default value of
-`perturb_mesh_3()` is used for this parameter, except for the time bound which is set to be
-equal to the refinement CPU time.
-
-- <b>`exude`</b> `parameters::exude()` and `parameters::no_exude()` are designed to
-trigger or not a call to `exude_mesh_3()` function and to override to set the
-parameters of this optimizer. If one parameter is not set, the default value of
-`exude_mesh_3()` is used for this parameter, except for the time bound which is set to be
-equal to the refinement CPU time.
-
-The optimization parameters can be passed in arbitrary order. If one parameter
-is not passed, its default value is used. The default values are
-`no_lloyd()`, `no_odt()`, `perturb()` and `exude()`.
-Note that whatever may be the optimization processes activated,
-they are always launched in the order that is a suborder
-of the following (see user manual for further
-details): *ODT-smoother*, *Lloyd-smoother*, *perturber*, and *exuder*.
-
-Beware that optimization of the mesh is obtained
-by perturbing mesh vertices and modifying the mesh connectivity
-and that this has an impact
-on the strict compliance to the refinement criteria.
-Though a strict compliance to mesh criteria
-is guaranteed at the end of the Delaunay refinement, this may no longer be true after
-some optimization processes. Also beware that the default behavior does involve some
-optimization processes.
-
-\sa `CGAL::make_mesh_3()`
-\sa `CGAL::parameters::manifold`
-\sa `CGAL::parameters::manifold_with_boundary`
-\sa `CGAL::parameters::non_manifold`
-\sa `CGAL::exude_mesh_3()`
-\sa `CGAL::perturb_mesh_3()`
-\sa `CGAL::lloyd_optimize_mesh_3()`
-\sa `CGAL::odt_optimize_mesh_3()`
-\sa `CGAL::parameters::exude`
-\sa `CGAL::parameters::no_exude`
-\sa `CGAL::parameters::perturb`
-\sa `CGAL::parameters::no_perturb`
-\sa `CGAL::parameters::lloyd`
-\sa `CGAL::parameters::no_lloyd`
-\sa `CGAL::parameters::odt`
-\sa `CGAL::parameters::no_odt`
-
-*/
-
-template <class C3T3, class MD, class MC>
-void refine_mesh_3(C3T3& c3t3,
-                   const MD& mesh_domain,
-                   const MC& mesh_criteria,
-                   parameters::internal::Lloyd_options lloyd = parameters::no_lloyd(),
-                   parameters::internal::Odt_options odt = parameters::no_odt(),
-                   parameters::internal::Perturb_options perturb = parameters::perturb(),
-                   parameters::internal::Exude_options exude = parameters::exude(),
-                   parameters::internal::Manifold_options manifold = parameters::non_manifold());
-
 namespace parameters {
```

Comment:
you cannot have twice the same header, you need to copy that doc `include/CGAL/refine_mesh.h` and remove that file